src/Components/WordTerm.js

```diff
@@ -1,25 +1,45 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
```

Comment:
Rather than importing each of these components on separate lines, you can actually use brackets to do it all in one!
```suggestion
import { Card, CardActions, CardContent, Button, Typography} from '@mui/material';
```